app/controllers/api/storage.php

```diff
@@ -244,6 +245,7 @@
                 'dateCreated' => \time(),
                 'dateUpdated' => \time(),
                 'name' => $name,
+                'permission' => $permission,
```

Comment:
I had to make changes to the Database routes when implementing collection/document permissions. Does anything else need to be added to the Storage controller? 
ref: https://github.com/appwrite/appwrite/pull/1480

Reply:
@kodumbeats I don't see the changes you made to the routes, can you point me in the right direction?